docs/datasheets/hopetwitter.md

```diff
@@ -0,0 +1,231 @@
+
+# HopeTwitter
+
+*Version*: 1.0.0
+
+*Homepage*: https://github.com/centre-for-humanities-computing/danish-foundation-models
+
+*license*: Not publicly available.
+
+---
+
+HopeTwitter consists of tweets collected from the Twitter API using a stopword list
+and consists of 32.5 million tweets across 538,398 unique users. HopeTwitter includes
+tweets from 2019-01-01 to 2021-04-30.
+
+HopeTwitter, have been filtered to only include Danish tweets. Similarly, HopeTwitter
```

Comment:
Maybe mention how the language was determined? Was it via Twitter's language identification algorithm?

Reply:
Agree. Better?